src/utils/flattenMetadataQueryResponse.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * @flow
+ * @file File for some simple dom utilities
+ * @author Box
+ */
+
+import omit from 'lodash/omit';
+import { getFileExtension } from './file';
+
+type ConvertMetadataParams = {
+    item: Object,
+    metadata: Object,
+};
+
+const convertISODateToSeconds = (dateString: string) => {
+    return dateString ? new Date(dateString).getTime() / 1000 : null;
+};
+
+const convertMetadata = (metadata: ConvertMetadataParams) => {
+    let md = {};
+
+    Object.keys(metadata).forEach(scope => {
+        Object.keys(metadata[scope]).forEach(templateKey => {
+            const nonconformingInstance = metadata[scope][templateKey];
+            const data = omit(nonconformingInstance, [
+                '$id',
+                '$parent',
+                '$type',
+                '$typeScope',
+                '$typeVersion',
+                '$version',
+            ]);
+
+            md = {
+                data,
+                id: nonconformingInstance.$id,
+                metadataTemplate: {
+                    type: 'metadata-template',
+                    templateKey,
+                },
+            };
+        });
+    });
+
+    return md;
+};
+
+const convertEntry = (entry: Object) => {
+    const { item, metadata } = entry;
+
+    return {
+        contentUpdated: convertISODateToSeconds(item.content_modified_at),
```

Comment:
It seems like some of the `item` properties can be destructured, as they're used more than once.